### DISCUSSION TITLE
Listing Recently Added/Updated Doumentation Pages

### DISCUSSION THREAD

**Discussion by ElementalCrisis** [facebook/docusaurus#9185] [Q&A] [answered]
Hello,

Is it possible to get a listing of recently added/updated documentation pages? Looking to create a section to show recent changes but didn't see anything in the docs about generating such a page.

Thank You!

**Comment by slorber** [comment, marked as answer]
We don't have that feature atm no.

Eventually we'll add a last mod attribute to our sitemaps, but maybe it's not what you want.

I think what you want could become possible thanks to React Server Components, see https://github.com/facebook/docusaurus/issues/9089

**Comment by ElementalCrisis** [comment]
Thanks for the info! 

A last mod attribute would work great since we're only using the docs portion of Docusaurus, looking forward to that future update! 

That issue also looks interesting, I'll keep an eye out on that as well.
